sci-electronics/splat/splat-1.4.2.ebuild

```diff
@@ -42,30 +44,28 @@ src_configure() {
 }
 
 src_compile() {
-	local CC=$(tc-getCC) CXX=$(tc-getCXX)
+	tc-export CXX
```

Comment:
The reason I mentioned to use `tc-export` was because "local CC" wouldn't be picked by emake, need either export or `emake CC=...`

It calls `$(CC)` which defaults to just `cc`, so now CC is missing.

Reply:
thanks, I noticed it also in the meantime